Mathlib/Topology/Bornology/Basic.lean

```diff
@@ -0,0 +1,358 @@
+/-
+Copyright (c) 2022 Jireh Loreaux. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jireh Loreaux
+
+! This file was ported from Lean 3 source module topology.bornology.basic
+! leanprover-community/mathlib commit 8631e2d5ea77f6c13054d9151d82b83069680cb1
+! Please do not edit these lines, except to modify the commit id
+! if you have ported upstream changes.
+-/
+import Mathlib.Order.Filter.Cofinite
+
+/-!
+# Basic theory of bornology
+
+We develop the basic theory of bornologies. Instead of axiomatizing bounded sets and defining
+bornologies in terms of those, we recognize that the cobounded sets form a filter and define a
+bornology as a filter of cobounded sets which contains the cofinite filter.  This allows us to make
+use of the extensive library for filters, but we also provide the relevant connecting results for
+bounded sets.
+
+The specification of a bornology in terms of the cobounded filter is equivalent to the standard
+one (e.g., see [Bourbaki, *Topological Vector Spaces*][bourbaki1987], **covering bornology**, now
+often called simply **bornology**) in terms of bounded sets (see `Bornology.ofBounded`,
+`IsBounded.union`, `IsBounded.subset`), except that we do not allow the empty bornology (that is,
+we require that *some* set must be bounded; equivalently, `∅` is bounded). In the literature the
+cobounded filter is generally referred to as the *filter at infinity*.
+
+## Main definitions
+
+- `Bornology α`: a class consisting of `cobounded : filter α` and a proof that this filter
```

Comment:
```suggestion
- `Bornology α`: a class consisting of `cobounded : Filter α` and a proof that this filter
```
Note that you can use `scripts/fix-comments.py File.Name.lean` to fix aligned names in comments.

Reply:
oh cool! I didn't know about that.